python/GangaDirac/Lib/Server/DiracCommands.py

```diff
@@ -113,22 +113,23 @@ def addFile(lfn, file, diracSE, guid):
     output(dirac.addFile(lfn, file, diracSE, guid))
 
 
-def getOutputSandbox(id, outputDir=os.getcwd(), oversized=True, pipe_out=True):
+def getOutputSandbox(id, outputDir=os.getcwd(), oversized=True, noJobDir=True, pipe_out=True):
     '''
     Get the outputsandbox and return the output from Dirac to the calling function
     id: the DIRAC jobid of interest
     outputDir: output directory locall on disk to use
     oversized: is this output sandbox oversized this will be modified
+    noJobDir: should we create a folder with the DIRAC job ID?
     output: should I output the Dirac output or should I return a python object (False)'''
-    result = dirac.getOutputSandbox(id, outputDir, oversized)
+    result = dirac.getOutputSandbox(id, outputDir, oversized, noJobDir)
     if result is not None and result.get('OK', False):
-        tmpdir = os.path.join(outputDir, str(id))
-        os.system('mv -f %s/* %s/.' % (tmpdir, outputDir))
-        os.system('rm -rf %s' % tmpdir)
-        ganga_logs = glob.glob('%s/*_Ganga_*.log' % outputDir)
 
-        if ganga_logs:
-            os.system('ln -s %s %s/stdout' % (ganga_logs[0], outputDir))
+        if not noJobDir:
+            tmpdir = os.path.join(outputDir, str(id))
+            os.system('mv -f %s/* %s/. ; rm -rf %s' % (tmpdir, outputDir, tmpdir))
+        
+        os.system('for file in $(ls %s/*_Ganga_*.log); do ln -s ${file} %s/stdout; break; done' % (outputDir, outputDir))
```

Comment:
Can we not do this and the line above using the Python API rather than falling back to shell commands?